tests/Parsers/MultipleChoiceQuestionParserTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Collecthor\SurveyjsParser\Tests\Parsers;
+
+use Collecthor\SurveyjsParser\Parsers\DummyParser;
+use Collecthor\SurveyjsParser\Parsers\MultipleChoiceQuestionParser;
+use Collecthor\SurveyjsParser\SurveyConfiguration;
+use Collecthor\SurveyjsParser\Variables\MultipleChoiceVariable;
+use PHPUnit\Framework\TestCase;
+use function iter\toArray;
+
+/**
+ * @covers \Collecthor\SurveyjsParser\Parsers\MultipleChoiceQuestionParser
+ * @uses \Collecthor\SurveyjsParser\Variables\MultipleChoiceVariable
+ * @uses \Collecthor\SurveyjsParser\Values\StringValueOption
+ * @uses \Collecthor\SurveyjsParser\Values\IntegerValueOption
+ * @uses \Collecthor\SurveyjsParser\Traits\GetDisplayValue
+ * @uses \Collecthor\SurveyjsParser\SurveyConfiguration
+ * @uses \Collecthor\SurveyjsParser\Variables\OpenTextVariable
+ */
+
+class MultipleChoiceQuestionParserTest extends TestCase
+{
+    public function testMultipleChoice(): void
+    {
+        $parent = new DummyParser();
+        $surveyConfiguration = new SurveyConfiguration();
+
+        $parser = new MultipleChoiceQuestionParser();
+
+        $questionConfig = [
+            'choices' => [
+                ['value' => 'a', 'text' => 'b'],
+                'c',
+                15,
+                ['value' => 16, 'text' => 'abc']
+            ],
+            'name' => 'q1',
+        ];
+
+        $variable = toArray($parser->parse($parent, $questionConfig, $surveyConfiguration));
+        self::assertInstanceOf(MultipleChoiceVariable::class, $variable[0]);
```

Comment:
This seems like a lazy and false way to get coverage.
You've confirmed that the parser doesn't crash, but you haven't tested anything related to the config you passed.

Reply:
Yes, good point.